dradis/dradis-ce
# Tylium - Card revision history

### Spec

We were already using PaperTrail for the :destroy event of the cards. This PR extends its use throughout the lifecycle of the model and adds the "Revision history" view to Cards#show in line with other models in the app.

### How to test
1. Add a card with some content
2. Confirm there is no "Revision history" link in the "Recent Activity" tab.
2. Make changes to the description
3. Confirm there is now a "Revision history" link that moves you to the diff view as expected.

### Check List
